ui/src/funding/contract.ts

```diff
@@ -0,0 +1,92 @@
+import type { BigNumberish, ContractTransaction, Signer } from "ethers";
+import {
+  Erc20Pool,
+  Erc20Pool__factory as PoolFactory,
+  ERC20,
+  ERC20__factory as Erc20Factory,
+} from "radicle-contracts/build/contract-bindings/ethers";
+
+// TODO(nuno): make the contract addresses configurable/env-dependant
+
+export const POOL_ADDRESS: string =
+  "0x37c10d847bf9e708add451bf2f80c1297d7aa691";
+
+export function pool(
+  signer: Signer,
+  address: string = POOL_ADDRESS
```

Comment:
It appears that this function is never called with a second argument. Can we remove it?